easybuild/easyconfigs/g/GMT/GMT-5.4.5-foss-2019a.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'CMakeMake'
+
+name = 'GMT'
+version = '5.4.5'
+local_gshhg_ver = '2.3.7'
+local_dcw_ver = '1.1.4'
+
+homepage = 'https://gmt.soest.hawaii.edu/'
+description = """GMT is an open source collection of about 80 command-line tools for manipulating
+ geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,
+ etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps
+ to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another
+ 40 more specialized and discipline-specific tools. """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+# 'https://gmt.soest.hawaii.edu/files/download?name=' needs browser with javascript magic
+source_urls = [
+    'ftp://ftp.soest.hawaii.edu/gmt',
+    'ftp://ftp.soest.hawaii.edu/gmt/legacy',
+    'ftp://ftp.soest.hawaii.edu/gshhg',
+    'ftp://ftp.soest.hawaii.edu/gshhg/legacy',
+    'ftp://ftp.soest.hawaii.edu/dcw',
+    'ftp://ftp.soest.hawaii.edu/dcw/legacy',
+]
+sources = [
+    '%(namelower)s-%(version)s-src.tar.gz',
+    # coastlines, rivers, and political boundaries
+    'gshhg-gmt-%s.tar.gz' % local_gshhg_ver,
+    # country polygons
+    'dcw-gmt-%s.tar.gz' % local_dcw_ver,
+]
+patches = ['GMT-5.1.2_netCDF.patch']
+checksums = [
+    '225629c7869e204d5f9f1a384c4ada43e243f83e1ed28bdca4f7c2896bf39ef6',  # gmt-5.4.5-src.tar.gz
+    '9bb1a956fca0718c083bef842e625797535a00ce81f175df08b042c2a92cfe7f',  # gshhg-gmt-2.3.7.tar.gz
+    '8d47402abcd7f54a0f711365cd022e4eaea7da324edac83611ca035ea443aad3',  # dcw-gmt-1.1.4.tar.gz
+    '2ebe26d55521fba8d0eae48f662611491f7cc0e489833bded11628e9a71f252f',  # GMT-5.1.2_netCDF.patch
+]
+
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [
+    ('PCRE', '8.43'),
+    ('GDAL', '3.0.0', '-Python-2.7.15'),
+    ('netCDF', '4.6.2'),
+    ('Ghostscript', '9.27'),
+    ('cURL', '7.63.0'),
+    ('zlib', '1.2.11'),
+]
+
+separate_build_dir = True
+
+configopts = '-DCOPY_GSHHG=TRUE -DGSHHG_ROOT=%%(builddir)s/gshhg-gmt-%s ' % local_gshhg_ver
+configopts += '-DCOPY_DCW=TRUE -DDCW_ROOT=%%(builddir)s/dcw-gmt-%s ' % local_dcw_ver
+
+sanity_check_paths = {
+    'files': [
+        ['bin/%s' % b for b in ['gmt', 'isogmt', 'gmtswitch', 'gmt_shell_functions.sh']],
```

Comment:
@lexming I am not sure it is ok to have a list in a list here. although the test succeeded, in the log I see that it checks for the first file only, `bin/gmt`.

Reply:
@smoors Indeed, this isn't how it should be:

```python
sanity_check_paths = {
    'files': ['bin/%s' % b for b in ['gmt', 'isogmt', 'gmtswitch', 'gmt_shell_functions.sh']] +
             ['lib64/libgmt.%s.%s' % (SHLIB_EXT, version)],
    'dirs': ['include/%(namelower)s', 'share']
}

Reply:
Thanks for the feedback. Fixed.